### PR TITLE
[Doc] add the ports that need to be opened in connectors docs

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by banmoy** [comment]
how about link xxx_port to StarRocks configuration documentation so that user can know what the port is?

File: docs/en/loading/Flink-connector-starrocks.md [hunk 1]
```diff
@@ -379,6 +379,10 @@ DISTRIBUTED BY HASH(`id`);
     ./bin/start-cluster.sh
     ```
 
+#### Network configuration
+
+Ensure that the machine where Flink is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```
Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:384]: image
It's better to remove "ports".

Comment (amber-create) [inline docs/en/loading/Flink-connector-starrocks.md:384]: image
remove "port"

File: docs/en/loading/Kafka-connector-starrocks.md [hunk 2]
```diff
@@ -35,6 +35,10 @@ Submit the Kafka connector into Kafka Connect:
 
   Currently, the Kafka connector is not uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a ZIP file and upload the ZIP file to Confluent Cloud.
 
+### Network configuration
+
+Ensure that the machine where Kafka is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```
Comment (amber-create) [inline docs/en/loading/Kafka-connector-starrocks.md:40]: image
remove them

File: docs/en/loading/Spark-connector-starrocks.md [hunk 3]
```diff
@@ -279,6 +279,10 @@ COMMENT "OLAP"
 DISTRIBUTED BY HASH(`id`);
 ```
 
+#### Network configuration
+
+Ensure that the machine where Spark is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```
Comment (amber-create) [inline docs/en/loading/Spark-connector-starrocks.md:284]: image
remove them

File: docs/en/unloading/Flink_connector.md [hunk 4]
```diff
@@ -85,19 +85,23 @@ You can also deploy Flink by following the instructions provided in [Flink docum
...
-2. If code debugging is needed, compile the Flink connector package to suit your business requirements.
+2. Place the Flink connector package you downloaded or compiled into the `lib` directory of Flink.
+
+3. Restart your Flink cluster.
 
-3. Place the Flink connector package you downloaded or compiled into the `lib` directory of Flink.
+### Network configuration
 
-4. Restart your Flink cluster.
+Ensure that the machine where Flink is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```
Comment (amber-create) [inline docs/en/unloading/Flink_connector.md:104]: image
remove them

File: docs/en/unloading/Spark_connector.md [hunk 5]
```diff
@@ -270,6 +270,10 @@ The processing logic of the underlying storage engine used by StarRocks cannot c
 
 The following examples assume you have created a database named `test` in your StarRocks cluster and you have the permissions of user `root`. The parameter settings in the examples are based on Spark Connector 1.1.0.
 
+### Network configuration
+
+Ensure that the machine where Spark is located can access the machine where the FE or FEs in a StarRocks cluster are located through the [`http_port`](../administration/management/FE_configuration.md#http_port) and [`query_port`](../administration/management/FE_configuration.md#query_port) ports, and access the machines where the BEs in a StarRocks cluster are located through the [`http_port`](../administration/management/BE_configuration.md#be_http_port) port.
```
Comment (amber-create) [inline docs/en/unloading/Spark_connector.md:275]: image
remove them

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Pull request merged**